Gemfile

```diff
@@ -1,14 +1,23 @@
 source "https://rubygems.org"
```

Comment:
The whole file is managed by our modulesync_config:
https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml#L24-L61

which results in:
https://github.com/voxpupuli/puppet-tea/blob/master/Gemfile

are any of the changes here still needed?